test/peliasDocGeneratorsTest.js

```diff
@@ -51,8 +51,24 @@ tape('create', function(test) {
         name: 'name 5',
         lat: 16.161616,
         lon: 61.616161,
+        place_type: 'county',
+        bounding_box: '-13.691314,49.909613,1.771169,60.847890'
+      },
+      6: {
+        id: 6,
+        name: 'name 6',
+        lat: 17.171717,
+        lon: 71.717171,
+        place_type: 'localadmin',
+        bounding_box: '-13.691314,49.909613,1.771169,60.847891'
+      },
+      7: {
+        id: 7,
+        name: 'name 7',
+        lat: 18.181818,
+        lon: 81.818181,
         place_type: 'locality',
-        bounding_box: '-13.691314,49.909613,1.771169,60.847890',
+        bounding_box: '-13.691314,49.909613,1.771169,60.847892',
```

Comment:
Since `locality` got bumped down, the test verifies that the bounding box of the base record is the one used and not any of the bounding boxes in the hierarchy.